elements/pfe-jump-links/src/pfe-jump-links-panel.scss

```diff
@@ -0,0 +1,10 @@
+::slotted(.pfe-jump-links__section--spacer) {
+  --pfe-jump-links-panel__section--spacer: calc(#{ var(--pfe-navigation--Height--actual) + var(--pfe-jump-links--nav-height) });
+  height: 100px;
+  margin-top: -100px;
+  display: block;
+  position: relative;
+  width: 0;
+  margin-top: calc( -1 * var(--pfe-jump-links-panel__section--spacer) );
```

Comment:
@starryeyez024 I'm having trouble getting the height and margin-top styles to tie into the custom properties from the demo.css file. Do lines 2, 8 & 9 look ok?

Reply:
added fix for line 2 (no interpolation needed)